packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -185,7 +185,7 @@ class DescopeWc extends BaseDescopeWc {
     // if there is no execution id we should start a new flow
     if (!executionId) {
       if (flowConfig.fingerprintEnabled && flowConfig.fingerprintKey) {
-        ensureFingerprintIds(flowConfig.fingerprintKey);
+        await ensureFingerprintIds(flowConfig.fingerprintKey, this.baseUrl);
```

Comment:
I understand why we need to await, but this will slow the flow

do you think we should maybe communicate this in the "Enable FP" toggle button in Flow Editor?